libs/store/src/lib/infrastructure/persistence/domain/base-classes/base-entity.ts

```diff
@@ -0,0 +1,49 @@
+import { Composite } from './composite';
+
+/**
+ * BaseEntityDTO that extends passed Entity DTO
+ * */
+export interface BaseEntityDTO {
+    id: number;
+    primaryKey?: string;
```

Comment:
You don't need to add the `primaryKey` property here. What I was trying to say was that the DTO from a service might not have `id` as its identifier, it may be some other field, like:
```ts
{
   uuid: '1a3dd8b8-bb2a-4e62-92f7-6f5f5cb2ea7' ,
   name: 'John',
   age: 24
}
```
Here `uuid` is the identifier for the DTO, and it is a string.